### PR TITLE
Docker init

### PR DESCRIPTION
**Pull request by oleksandrtarar**
Add Dockerfile and docker-compose.yml

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:16.04
```
Comment (serge30) [inline Dockerfile]: I would prefer to use `python:2.7` image, as it designed to run python code and already includes Python 2.7 and `pip`.

File: README.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#    1. docker build -t pyramid .
```
Comment (serge30) [inline README.txt]: Please use MarkDown format for documentation and file name has to be `README.md`. Add sections describing setup and info about ports available. Other info might be added during development.

Is step `docker build -t pyramid .` needed here? I believe `docker-compose up` performs it.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+RUN apt-get update -y && \
+    apt-get install -y python-pip python-dev && \
+    pip install --upgrade pip setuptools
+
+COPY ./requirements.txt /requirements.txt
+
+WORKDIR /app
+
+RUN pip install -r /requirements.txt
```
Comment (serge30) [inline Dockerfile]: Please add `EXPOSE 6543` statement. It's not mandatory, but it helps to understand which ports container will be listen to.

File: docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+version: '2'
```
Comment (serge30) [inline docker-compose.yml]: Any reasons to do not use latest format version? I think version `3` might be good here.

File: docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+          POSTGRES_DB: eventme
+        volumes:
+          - ~/.eventme-postgres:/var/lib/postgresql/data
+    adminer:
+        image: adminer
+        restart: always
+        depends_on:
+            - db
+        ports:
+          - 9000:8080
```
Comment (serge30) [inline docker-compose.yml]: Please configure you editor to add empty line to the end of file. It's common practice.

**Review by serge30 — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 6]
```diff
@@ -0,0 +1,12 @@
+FROM python:2.7
+
+RUN apt-get update -y && \
+    pip install --upgrade pip setuptools
```
Comment (serge30) [inline Dockerfile]: As `python` image already has `pip` I think we may skip step to update deb packages, pip and setuptools.

File: README.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
+### Docker ###
+
+    Open your favorite Terminal and run these commands.
```
Comment (serge30) [inline README.md]: Please remove indentation, because it makes formatting incorrect. You may check MarkDown formatting using https://dillinger.io/

**Review by serge30 — CHANGES_REQUESTED** [review]

**Review by serge30 — APPROVED** [review]
Looks good! You can merge it.

File: docker-compose.yml (hunk 5 above)
Comment (oleksandrtarar) [reply docker-compose.yml]: I already have 2 empty lines at the end of file, but "github" doesn't show them.

**Pull request merged**